src/router/bundle/bundleGenerator.ts

```diff
@@ -129,4 +129,51 @@ export default class BundleGenerator {
         response.entry = entries;
         return response;
     }
+
+    static generateBatchBundle(baseUrl: string, bundleEntryResponses: BatchReadWriteResponse[]) {
+        const id = uuidv4();
+        const response = {
+            resourceType: 'Bundle',
+            id,
+            type: 'batch-response',
+            link: [
+                {
+                    relation: 'self',
+                    url: baseUrl,
+                },
+            ],
+            entry: [],
+        };
+
+        const entries: any = [];
+        bundleEntryResponses.forEach((bundleEntryResponse) => {
+            let status = '200 OK';
+            if (bundleEntryResponse.operation === 'create') {
+                status = '201 Created';
+            } else if (
+                ['read', 'vread'].includes(bundleEntryResponse.operation) &&
```

Comment:
I thought vread is not supported? Also why would `isEmpty(bundleEntryResponse.resource)` map to 403 instead of 404?